### DISCUSSION TITLE
How do I get rid of random numbers in the listing?

### DISCUSSION THREAD

**Discussion by gerroon** [AdguardTeam/AdGuardHome#5403] [Q&A]
Hi

I have a lot of these type of queries in my listing. How do I get rid of them. I am not super techy aboutt his stuff.

![chrome_3TX7BvV2mv](https://user-images.githubusercontent.com/8519469/214715784-59616622-85cc-4593-87b0-7dd07bf1fba7.jpg)

**Comment by ammnt** [comment]
@gerroon, its okay:
https://en.wikipedia.org/wiki/.arpa

No need to do something here🙂

**Reply by gerroon** [reply]
> @gerroon, its okay: https://en.wikipedia.org/wiki/.arpa
> 
> No need to do something hereslightly_smiling_face

Well, these pollute my listings so badly, I cant see anything else. I was hoping to be able to just filter them out.

**Comment by RonKhondji** [comment]
I don't really know if it'll help but you can try turning of: 'Enable reverse resolving of clients ip addresses' under the dns settings.
